css/about.css

```diff
@@ -0,0 +1,388 @@
+  padding: 1rem;
+}
+
+.about-help-details {
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+}
+
+.about-help-text {
+  color: grey;
+  text-align: center;
+  font-size: 1.2rem;
+  font-weight: 700;
+}
+
+.about-help-link {
+  display: inline-block;
+  width: 100%;
+  text-align: center;
+  color: rgba(34, 34, 34, 0.984);
+  font-weight: 600;
+  text-decoration: underline;
+}
+
+@media screen and (min-width: 768px) {
+  .about-image {
+    height: 100%;
+    padding-top: 8rem;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .about-wrapper {
+    gap: 4rem;
+    max-width: 75rem;
+  }
+
+  .about-title {
+    max-width: 70rem;
+    margin: 0 auto;
+    font-size: 4rem;
+    padding: 0 2rem;
+  }
+
+  .about-desc {
+    border-radius: 0.5rem;
+  }
+}
+
+/* logo section */
+.logo-wrapper {
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 3rem;
+  margin: 2rem 0;
+}
+
+.logo-title {
+  color: #272b30;
+  font-size: 2rem;
+  font-weight: 900;
+  text-align: center;
+}
+
+.logo-text {
+  color: grey;
+  text-align: center;
+}
+
+.logo-image {
+  padding: 2rem 1rem;
+  border-top: 1px solid #d3d3d3;
+  border-bottom: 1px solid #d3d3d3;
+  max-width: 21.875rem;
+}
+
+@media screen and (min-width: 768px) {
+  .logo-text {
+    max-width: 70rem;
+  }
+}
+
+/* summit event section */
+.summit-events-wrapper {
+  margin-bottom: 2rem;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.summit-event-title {
+  color: #272b30;
+  font-size: 2rem;
+  font-weight: 900;
+  text-align: center;
+  padding: 0 1rem;
+}
+
+.summit-event-text {
+  color: gray;
+  text-align: center;
+  margin: 1rem 0;
+  padding: 0 1rem;
+}
+
+.summit-events {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 2rem;
+}
+
+.summit-event-link-one,
+.summit-event-link-two {
+  padding: 0 1rem;
+  width: 21.875rem;
+  height: 18.75rem;
+  background-image: url("../assests/summit-event-one.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  color: white;
+  font-size: 2rem;
+  font-weight: 700;
+}
+
+.summit-event-link-one > span:first-child,
+.summit-event-link-two > span:first-child {
+  text-transform: uppercase;
+  font-weight: 900;
+}
+
+.summit-event-link-two {
+  background-image: url("../assests/summit-event-two.png");
+}
+
+@media screen and (min-width: 768px) {
+  .summit-event-text {
+    max-width: 70rem;
+  }
+
+  .summit-events {
+    flex-direction: row;
+  }
+}
+
+/* footer section */
+.footer {
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+  padding: 1rem 0;
+  background-color: white;
+}
+
+.footer-content {
+  margin-left: 1rem;
+}
+
+@media screen and (min-width: 768px) {
+  .footer {
+    background-color: #272b30;
+  }
```

Comment:
- [ ] Good job on changing the background color! 👍👍 However, I noticed that the desktop version now has a dark (black) text color, whereas the design guidelines recommend a light color for the desktop version. ✨Previously it also had a light color (desktop version). Kindly adjust this to align with the design guidelines mentioned in the project requirements.


Your design | Given design 
---| ---|
![image](https://github.com/Titus-Kiplagat/capstone-project-htmlcss-js/assets/106119105/82459dfd-263e-46e4-8f4f-d9ece6ad17d9) | ![image](https://github.com/Titus-Kiplagat/capstone-project-htmlcss-js/assets/106119105/0e5ead4d-1bcf-46dc-ba3f-c24985470e1f)